node/building_blocks.re

```diff
@@ -55,6 +55,47 @@ let is_current_producer = (state, ~key_hash) => {
   Some(current_producer.address == key_hash);
 };
 
+/** Blocks that change the state root epoch before
+    this interval in seconds has passed are not signable.
+
+    TODO: verify reasonable times for this with benchmarking.
+          currently set to [state_root_min_timeout] as a reasonable
+          default. */
+let minimum_signable_time_between_epochs = 60.0;
+/** Blocks that do not change the state root epoch after
+    this interval has passed are not signable.
+
+    TODO: verify reasonable times for this with benchmarking.
+    Currently set to [minimum_signable_time_between_epochs] + 10
+    as a reasonable default. */
+let maximum_signable_time_between_epochs = 70.0;
+
+/* Returns true if the block's state root hash and has not
```

Comment:
> if the block's state root hash and has not

is "and" not needed here? I'm confused a bit

Reply:
You are correct. I've fixed the comment.